afqinsight/nn/tests/test_pt_models.py

```diff
@@ -0,0 +1,204 @@
+import numpy as np
+import pytest
+import torch
+
+from afqinsight import AFQDataset
+from afqinsight.nn.pt_models import (
+    blstm1_pt,
+    blstm2_pt,
+    cnn_lenet_pt,
+    cnn_resnet_pt,
+    cnn_vgg_pt,
+    lstm1_pt,
+    lstm1v0_pt,
+    lstm2_pt,
+    lstm_fcn_pt,
+    mlp4_pt,
+)
+from afqinsight.nn.utils import prep_pytorch_data
+
+
+@pytest.fixture
+def device():
+    """Fixture to set up the computing device."""
+    if torch.backends.mps.is_available():
+        return torch.device("cpu")
+    return torch.device("cuda" if torch.cuda.is_available() else "cpu")
+
+
+@pytest.fixture
+def dataset():
+    """Fixture to load the AFQ dataset."""
+    return AFQDataset.from_study("hbn")
+
+
+@pytest.fixture
+def data_loaders(dataset):
+    """Fixture to prepare PyTorch datasets and data loaders."""
+    torch_dataset, train_loader, test_loader, val_loader = prep_pytorch_data(dataset)
+    return torch_dataset, train_loader, test_loader, val_loader
+
+
+@pytest.fixture
+def data_shapes(data_loaders):
+    """Fixture to compute shapes for input and target tensors."""
+    torch_dataset = data_loaders[0]
+    gt_shape = torch_dataset[0][1].size()[0]
+    sequence_length = torch_dataset[0][0].size()[0]  # 48
+    in_channels = torch_dataset[0][0].size()[1]  # 100
+    return gt_shape, sequence_length, in_channels
+
+
+def run_pytorch_model(
+    model,
+    device,
+    data_loaders,
+    n_epochs=20,
+    permute=False,
+):
+    """
+    Smoke testing on PyTorch models to ensure it trains and tests correctly.
+
+    Args:
```

Comment:
This should follow the [numpy docstring standard]()
```suggestion
    Parameters
    ----------
```